roles/ansible_tower/tasks/main.yaml

```diff
@@ -30,3 +30,13 @@
 
 - name: Set (httpd_can_network_connect_db) flag on and keep it persistent across reboots
   seboolean: name=httpd_can_network_connect_db state=yes persistent=yes
+
+- name: Setup proot to allow access to /etc/tower/
+  lineinfile:
+    dest: /etc/tower/settings.py
+    backrefs: yes
+    regexp: "^({{ item.option }})( *)="
+    line: '\1\2 = {{ item.value }}'
+  with_items: config_changes | default([])
```

Comment:
Just seeing this... What we really should do in this case is to set the boolean parameter for the default filter `| default([], true)` Otherwise an empty string could cause the with_items to still fail.  I'm not sure if it is applicable in this case, but I have started trying to use it defensively in all cases that make sense.
http://jinja.pocoo.org/docs/dev/templates/#default